.gitpod.yml

```diff
@@ -0,0 +1,14 @@
+tasks:
+  - name: Configure nerdctl
+    command: mkdir -p /home/gitpod/.config/nerdctl && echo 'snapshotter = "native"' > /home/gitpod/.config/nerdctl/nerdctl.toml
+  - name: Run containerd
+    command: bash -c "curl https://raw.githubusercontent.com/containerd/containerd/main/script/setup/install-cni | bash" && sudo containerd
+  - name: Watch nerdctl
+    before: npm install --global nodemon
+    command: nodemon --watch './**/*.go' --ext go --signal SIGTERM --exec 'make binaries && make install && sudo chown root $BINDIR/nerdctl && sudo chmod +s $BINDIR/nerdctl'
```

Comment:
Any reason not to use rootless mode?

Reply:
I tried
```
$ containerd-rootless-setuptool.sh install
[ERROR] Needs systemd (systemctl --user)
```

Reply:
Can you just run `containerd-rootless.sh` ?

Reply:
```
$ containerd-rootless.sh
/home/gitpod/.local/bin/containerd-rootless.sh: 118: exec: rootlesskit: not found
```

If you want to play with this setup yourself: https://gitpod.io/#https://github.com/containerd/nerdctl/pull/1469

Reply:
rootlesskit should not be necessary, as processes within the workspace already run in a properly set up user namespace.